templates/mass_eigenstates.cpp.in

```diff
@@ -462,18 +462,36 @@ double CLASSNAME::G0(double p, double m1, double m2) const noexcept
    ).real();
 }
 
-/// derivative of B0(p^2, ma^2, mb^2) w.r.t. p^2
-double CLASSNAME::DB0(double p2, double m2a, double m2b) const noexcept
+double CLASSNAME::DH0(double p, double m1, double m2) const noexcept
```

Comment:
Please add a brief comment her that it is the derivative of H0 w.r.t. p^2 and the arguments are of dimension mass squared.

Reply:
Done in 46963d81122636e42cb7c82d7f26193087090e7d.